spec/controllers/api/v1/project_preferences_controller_spec.rb

```diff
@@ -137,4 +137,34 @@
 
     it_behaves_like "is creatable"
   end
+
+  describe "#update_settings", :focus do
+    let!(:project) { create(:project, owner: authorized_user) }
+    let!(:upp) { create(:user_project_preference, project: project) }
+    let(:settings_params) do
+      {
+        user_id: upp.user_id,
+        project_id: project.id,
+        settings: { workflow_id: 1234 }
+      }
+    end
+
+    let(:scopes) { %w(project_preference) }
```

Comment:
Without that let(:scopes), it's a 422. Remove the "scopes:" from the default_request and it's a 403. Remove default_request entirely and it's a 422.